oxygen-account/src/main/java/com/oxygenxml/account/exception/EmailException.java

```diff
@@ -0,0 +1,23 @@
+package com.oxygenxml.account.exception;
+
+import org.springframework.core.NestedCheckedException;
+
+/**
+ * Custom checked exception for email-related errors.
+ */
+public class EmailException extends NestedCheckedException  {
+    /**
```

Comment:
De ce ati ales sa extindeti din `NestedCheckedException`? Ar trebui sa extindeti din 'java.lang.Exception'.

La sedinta am zis ca exceptia voastra sa fie un "checked exception". In Java exista 2 categorii de exceptii: "checked" si "unchecked" exceptions. 
Va rog sa va documentati putin pe tema asta.